src/App/Fossa/VSI/IAT/Types.hs

```diff
@@ -15,12 +15,16 @@ import Data.Aeson (
   withObject,
   (.:),
  )
+import Data.String.Conversion (toString)
 import Data.Text (Text)
 
 -- | Fingerprint uniquely idenfies a file, derived from its content.
 -- Fingerprints are backed by base16 representations of underlying data.
 newtype Fingerprint = Fingerprint {unFingerprint :: Text}
 
+instance Show Fingerprint where
+  show f = toString $ unFingerprint f
+
```

Comment:
[nit] let's `deriving Show` on `Fingerprint` instead

(hopefully we're not using this `Show` instance anywhere directly -- `Show` by convention is primarily for debug)

Reply:
Oops, I added this for debugging and forgot to remove it. Will do.